partitioner-config-service-api/src/main/proto/org/hypertrace/partitioner/config/service/v1/partitioner_config_service.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+package org.hypertrace.partitioner.config.service.v1;
+
+option java_multiple_files = true;
+
+
+service PartitionerConfigService {
+
+  rpc GetPartitionerProfile(GetPartitionerProfileRequest) returns (GetPartitionerProfileResponse) {}
+
+  rpc PutPartitionerConfig(PutPartitionerConfigRequest) returns (PutPartitionerConfigResponse) {}
+
+  rpc GetAllPartitionerConfig(GetAllPartitionerConfigRequest) returns (GetAllPartitionerConfigResponse) {}
```

Comment:
micro-nit: put get and get all next to each other

Reply:
also config was replaced with profiles. GetAll, let's change to plural get - that way, if we ever need to add the ability to filter in the future it slots in cleanly.

```suggestion
  rpc GetPartionerProfiles(GetPartionerProfilesRequest) returns (GetPartionerProfilesResponse) {}
```

Reply:
done